test/Masa.Contrib.BasicAbility.Tsc.Test/Service/MetricServiceTests.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Masa.Contrib.BasicAbility.Tsc.Tests.Service;
+
+[TestClass]
+public class MetricServiceTests
+{
+    private ITscClient _client;
+
+    [TestInitialize]
+    public void Initialize()
+    {
+        IServiceCollection service = new ServiceCollection();
+        service.AddTscClient("https://localhost:6324/");
+        _client = service.BuildServiceProvider().GetService<ITscClient>() ?? default!;
```

Comment:
If _client is not null, then
`_client = service.BuildServiceProvider().GetRequiredService<ITscClient>();`

Otherwise _client should be a nullable type